patches/extensions-browser-BUILD.gn.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/extensions/browser/BUILD.gn b/extensions/browser/BUILD.gn
+index 42ddb608b355cf1b276fd12a04bded01658fdc11..3bb4d78cda9ea7099459f73f770daf984e4c024c 100644
+--- a/extensions/browser/BUILD.gn
++++ b/extensions/browser/BUILD.gn
+@@ -41,6 +41,9 @@ source_set("browser_context_keyed_service_factories") {
+ jumbo_source_set("browser_sources") {
+   visibility = [ "./*" ]
+ 
++  # //brave/components/greaselion/browser uses this.
```

Comment:
this patch is not needed at all, but don't add comments or extra spaces to patches

Reply:
This patch file is removed after adding //extensions/browser instead of //extensions/browser:browser_sources.